.github/workflows/documentation.yml

```diff
@@ -0,0 +1,48 @@
+name: Documentation
+
+on:
+  push:
+    branches: main
+    tags: '*'
```

Comment:
this is where we need to decide `when` exactly our doc page needs to be rebuilt. Only for a new tag? Everytime main is updated?

Reply:
I'd say whenever we update main.

Reply:
There is actually a clear answer to this, once we arrive at our "ideal" scenario: ultimately, we want the multiversion support in our docu. We would then have versions for each tag + one latest version. Right now, it doesn't matter that much. To see the effect of our changes, I'd probably also say: lets do "main" now, and then add the multiversion support once ready

Reply:
fine with that, but then tag trigger should be removed, it could build an older version than HEAD

Reply:
Ok, will remove.